### ISSUE TITLE
Mac Support

### ISSUE THREAD

**Issue by MightySCollins** [maidstone-hackspace/dockyard#6]
It would be good to have Mac support but I can see there being a problem with it using a virtual machine.
http://stackoverflow.com/questions/34256645/connecting-to-docker-machine-via-docker-py-on-osx might help and there is an issue open to add in settings for the docker client url #5

**Comment by apepindev** [comment]
Yes I have been attempting this but to no great extent. I also came across that thread on SO but I still had no luck with getting the containers from the machine.
